servers/rendering/rasterizer_rd/shaders/tonemap.glsl

```diff
@@ -23,7 +23,8 @@ layout(location = 0) in vec2 uv_interp;
 layout(set = 0, binding = 0) uniform sampler2D source_color;
 layout(set = 1, binding = 0) uniform sampler2D source_auto_exposure;
 layout(set = 2, binding = 0) uniform sampler2D source_glow;
-layout(set = 3, binding = 0) uniform sampler3D color_correction;
+layout(set = 3, binding = 0) uniform sampler2D color_correction_1d;
+layout(set = 4, binding = 0) uniform sampler3D color_correction_3d;
```

Comment:
Mobile devices can only have 4 uniform sets. 

The easiest way for this to work is to use a new shader version as we discussed on IRC. A shader version is created by adding an ``ifdef`` statement. 

If you want I can make a PR to your branch, just let me know.